test.e2e/spec/services.spec.js

```diff
@@ -1251,6 +1251,76 @@ define(['angular', 'given', 'util'], function(angular, given, util) {
       });
     });
 
+    describe('invalid namespaceCommonModels:true', function() {
+      var $injector;
+      before(function() {
+        return given.servicesForLoopBackApp(
+          {
+            models: {
+              Product: {
+                properties: {
+                  name: 'string',
+                  price: { type: 'number' },
+                },
+              },
+            },
+            name: 'lbServices',
+            namespaceCommonModels: true,
+            namespaceDelimiter: '_',
```

Comment:
The `describe` block says "invalid namespaceCommonModels:true", but I don't see which part of the configuration is invalid. What am I missing?